tests/tests.js

```diff
@@ -287,6 +287,100 @@
 			'{"hello":"world","\\uD83D\\uDCA9":"foo","pile":"\\uD83D\\uDCA9"}',
 			'`toJSON` methods are not called when `json: false`'
 		);
+		equal(
+			jsesc('\u2192\xE9', {
+				'lowercaseHex': true
+			}),
+			'\\u2192\\xe9',
+			'Alphabetical hexadecimal digits are lowercase when `lowercaseHex: true`'
+		);
+		equal(
+			jsesc('\u2192\xe9', {
+				'lowercaseHex': true,
+				'json': true
+			}),
+			'"\\u2192\\u00e9"',
+			'Alphabetical hexadecimal digits are lowercase when `lowercaseHex: alse` and `json: true`'
+		);
+		equal(
+			jsesc('test', {
+				'lowercaseHex': true,
+				'escapeEverything': true
+			}),
+			'\\x74\\x65\\x73\\x74',
```

Comment:
Maybe use a string like [`'I\xF1t\xEBrn\xE2ti\xF4n\xE0liz\xE6ti\xF8n\u2603\uD83D\uDCA9'`](https://mathiasbynens.be/notes/javascript-unicode#poo-test)?